analytics/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM rocker/r-ver:4.0.3
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+  	apt-utils \
+  	ed \
```

Comment:
Something's still using ed?

Reply:
Good catch, I am not sure where that `RUN` block came from, but this image does not need any of those packages.